src/cypress/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "scripts": {
+    "tests": "npx cypress run --spec 'cypress/integration/demo.feature' --browser=chrome"
```

Comment:
Let's remove the `--spec` and expand these scripts.
For example -
`test:cmd` - run cypress run for all tests
`test:interactive` - open cypress interactive window

Reply:
fixed in latest commit